pages/common/git-revert.md

```diff
@@ -5,7 +5,7 @@
 
 - Revert the most recent commit:
 
-`git revert {{@}}`
+`git revert HEAD`
```

Comment:
I feel that `HEAD` should be tokenized
```suggestion
`git revert {{HEAD}}`
```
@CleanMachine1 ?

Reply:
I agree :+1: